### PR TITLE
add 10% prepayment amount feature

### PR DESCRIPTION
**Pull request by DevFojo**

### CODE REVIEW

File: ke-legacy/main.js [hunk 1]
```diff
@@ -268,15 +268,17 @@ var ValNationalID = function(input){
...
-    //else {return false}
+
+var IsPrePayTrialDistrict = function(districtName){
+    return districtName === 'Kipkelion' || districtName === 'Chwele';
 };
+
 var GetPrepaymentAmount = function(client){
-    var ClientDistrict = client.DistrictName.toLowerCase();
-    var prepay = IsPrePayTrialDistrict(ClientDistrict);
+    if (client.DistrictName === 'Kipkelion' || client.DistrictName === 'Chwele'){
```
Comment (nyabongo) [inline ke-legacy/main.js:277]: This looks link a duplicate of  the `IsPrePayTrialDistrict` function?

**Review by Bahatiroben — DISMISSED** [review]
looks good.

File: ke-legacy/main.js [hunk 2]
```diff
@@ -268,27 +268,26 @@ var ValNationalID = function(input){
...
-        if (PrePayCursor.hasNext()) {
-            var PrePayRow = PrePayCursor.next();
-            return PrePayRow.vars.prepaymentamount;}
-        else {sendEmail('charles.lipeyah@oneacrefund.org', 'Prepayment amount not found', 'Prepayment amount not uploaded from client with acc num :'+ client.AccountNumber+ ' in district: '+client.DistrictName);
-            return 'Error';
-        }
-    }
-    else {return 500;}
+    return client.BalanceHistory[0].TotalCredit * 0.1;
+    // var prepay = IsPrePayTrialDistrict(client.DistrictName);
```
Comment (nyabongo) [inline ke-legacy/main.js:278]: won't this affect all the clients? the ticket says it should be  for only the 2 districts shouldn't this be
```suggestion
   var prepay = IsPrePayTrialDistrict(client.DistrictName);
   return prepay ?client.BalanceHistory[0].TotalCredit * 0.1 : 500;
```

File: ke-legacy/main.js [hunk 3]
```diff
@@ -268,27 +268,26 @@ var ValNationalID = function(input){
     if (NumChar == 7 || NumChar == 8){return true;}
     else {return false;}
 };
-var IsPrePayTrialDistrict = function (){
-    return false;
-    //districtname = districtname.toLowerCase();
-    //if (districtname == "nyando" || districtname == "kipkelion" || districtname == "chwele"){return true}
-    //else {return false}
+
+var IsPrePayTrialDistrict = function(districtName){
```
Comment (shejasabin) [inline ke-legacy/main.js:272]: To avoid confusions delete this maybe? I think it's now in populate menu

File: ke-legacy/main.js (hunk 2 above)
Comment (DevFojo) [reply ke-legacy/main.js]: We are already doing the check in utils/menus/populateMenu.js
https://github.com/one-acre-fund/ussd-sms/blob/ce8d55035eb0b96e42fe46bee6eb2a371eaee3ee/ke-legacy/utils/menus/populate-menu/populateMenu.js#L8-L11

**Review by nyabongo — APPROVED** [review]

**Review by Bahatiroben — APPROVED** [review]

**Pull request merged**